static/css/siglens.css

```diff
@@ -5603,6 +5603,38 @@ ul.metrics-dropdown li, ul.metrics-dropdown li a{
 .icon {
   visibility: hidden;
 }
+.error-message ,#metric-error{
+    visibility: hidden;
```

Comment:
1. Make the `display:none` instead of `visibility: hidden` as visibility property takes up the space for the element.
2. Fix the dark mode.